django_irods/storage.py

```diff
@@ -334,11 +334,16 @@ def _list_files(self, path):
         fname_list = []
         fsize_list = []
 
+        default_resc = getattr(settings, 'IRODS_DEFAULT_RESOURCE', None)
```

Comment:
take advantage of `IRODS_DEFAULT_RESOURCE` that we already have defined in our build/deploy environments